packages/@sanity/dashboard/src/widgets/projectUsers/ProjectUsers.js

```diff
@@ -85,64 +98,74 @@ class ProjectUsers extends React.PureComponent {
 
     if (error) {
       return (
-        <div>
-          Something went wrong while fetching data. You could{' '}
-          <a className={styles.retry} onClick={this.handleRetryFetch} title="Retry users fetch">
-            retry
-          </a>
-          ..?
-        </div>
+        <DashboardWidget header="Project users">
+          <Box padding={4}>
+            <Text>
+              Something went wrong while fetching data. You could{' '}
+              <a
+                onClick={this.handleRetryFetch}
+                title="Retry users fetch"
+                style={{cursor: 'pointer'}}
+              >
+                retry
+              </a>
+              ..?
+            </Text>
+          </Box>
+        </DashboardWidget>
       )
     }
 
     return (
-      <div className={styles.container}>
-        <header className={styles.header}>
-          <h2 className={styles.title}>Project users</h2>
-        </header>
-
+      <DashboardWidget
+        header="Project users"
+        footer={
+          <Card padding={4} as="a" href={isLoading ? undefined : getInviteUrl(project.id)}>
+            <Text align="center" weight="semibold">
+              Invite members
+            </Text>
+          </Card>
+        }
+      >
         {isLoading && (
-          <List className={styles.list}>
-            <Spinner center message="Loading items…" />
-          </List>
+          <Box paddingY={5} paddingX={2}>
+            <Stack space={4}>
+              <Text align="center" muted size={1}>
+                <Spinner />
+              </Text>
+              <Text align="center" size={1} muted>
+                Loading items...
+              </Text>
+            </Stack>
+          </Box>
         )}
 
         {!isLoading && (
-          <List className={styles.list}>
+          <Stack space={2} padding={2}>
             {users.map((user) => {
               const membership = project.members.find((member) => member.id === user.id)
               const media = membership.isRobot ? (
-                <RobotIcon className={styles.profileImage} />
+                <Text size={3}>
+                  <RobotIcon />
+                </Text>
               ) : (
-                <div className={styles.avatar}>
-                  {user.imageUrl && <img src={user.imageUrl} alt={user.displayName} />}
-                </div>
+                <AvatarWrapper tone="transparent">
+                  {user?.imageUrl && <img src={user.imageUrl} alt={user?.displayName} />}
+                </AvatarWrapper>
```

Comment:
Would it be possible to use the [UserAvatar](https://github.com/sanity-io/sanity/blob/next/packages/@sanity/base/src/components/UserAvatar.tsx) component here instead?

```tsx
import {UserAvatar} from '@sanity/base/components'

<UserAvatar user={user} />
// or
<UserAvatar userId={userId} />
```

Reply:
I tried that, but the styling of the UserAvatar was breaking for some reason. I guess it's some styling inherited from the parent component of `DefaultPreview` (the soon outdated one) that is causing this.

<img width="72" alt="Screenshot 2021-07-08 at 15 12 45" src="https://user-images.githubusercontent.com/15094168/124927462-f7fa5f00-dffe-11eb-889b-5c74c0b0d618.png">

Reply:
Interesting. Lets make a note to revisit this later, would be good to cut down on the number of different avatars throughout the studio